fe/fe-core/src/main/java/com/starrocks/external/hive/HdfsFileBlockDesc.java

```diff
@@ -8,6 +8,7 @@
     private long[] replicaHostIds;
     private long[] diskIds;
     private HiveMetaClient metaClient;
+    private boolean splittable;
```

Comment:
Better to put splittable in HdfsFileDesc or HivePartition (if all files in one partition are same format)